src/Support/Support/Az.Support.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.0'
+ModuleVersion = '1.0.1'
```

Comment:
please do not change version, Azure-PowerShell team will bump it during each release